starport/templates/app/stargate/.github/workflows/release.yml

```
@@ -0,0 +1,51 @@
+# This workflow is useful if you want to automate the processs of:
+#
+# a) Creating a new prelease when you push a new tag with a "v" prefix (version).
+#
+#    This type of prerelease is meant to be used for production: alpha, beta, rc, etc. type of releases.
+#    After the prerelease is created, you need to make your changes on the release page at the relevant
+#    Github page and publish your release.
+#
+# b) Creating/updating the "latest" prerelease when you push to your default branch.
+#
+#    This type of prelease is useful to make your bleeding-edge binaries avaiable to advanced users.
+#
+# The workflow will not run if there is no tag pushed with a "v" prefix and no change pushed to your 
+# default branch.
+on: push
+
+jobs:
+  might_release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        
+      - name: Prepare Release Variables 
+        id: vars
+        uses: tendermint/starport/actions/release/vars@develop
+
+      - name: Issue Release Assets 
+        uses: tendermint/starport/actions/cli@develop
+        if: ${{ steps.vars.outputs.should_release == 'true' }}
+        with:
+          args: build --release --release.prefix ${{ steps.vars.outputs.tarball_prefix }} -t linux:amd64 -t linux:arm64 -t darwin:amd64 -t darwin:arm64
+
+      - name: Delete the "latest" Release
+        uses: dev-drprasad/delete-tag-and-release
```

```suggestion
        uses: dev-drprasad/delete-tag-and-release@v0.2.0
```

Otherwise, I'm getting `The workflow is not valid. .github/workflows/release.yml (Line: 35, Col: 15): Expected format {org}/{repo}[/path]@ref. Actual 'dev-drprasad/delete-tag-and-release' Input string was not in a correct format.`